packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -148,7 +158,42 @@ const useBigNumberConfig = (
               compact: bigNumberStyle,
           });
 
-    const showStyle = isNumber && (!isField(item) || item.format !== 'percent');
+    const formatValues = {
+        RAW: 'raw',
+        PERCENTAGE: 'percentage',
+    };
+    const [showComparison, setShowComparison] = useState<boolean>(false);
+    const [comparisonFormat, setComparisonFormat] = useState<string>(
+        formatValues.RAW,
+    );
+
+    const calculateComparisonValue = (a: number, b: number, format: string) => {
+        const rawValue = Math.round(a - b);
+        switch (format) {
+            case formatValues.PERCENTAGE:
+                return Math.round(((a - b) / b) * 100);
+
+            case formatValues.RAW:
+                return rawValue === 0 ? 'No change' : rawValue;
+            default:
+                return rawValue;
+        }
+    };
+
+    const comparisonValue =
+        firstRowValueRaw &&
+        secondRowValueRaw &&
+        isSecondRowValueNumber &&
+        isFirstRowValueNumber
+            ? calculateComparisonValue(
+                  Number(firstRowValueRaw),
+                  Number(secondRowValueRaw),
+                  comparisonFormat,
+              )
+            : 'Comparison not applicable';
```

Comment:
@PriPatel @TuringLovesDeathMetal 
In the ticket, it mentions displaying the text `n/a` when we can't make a comparison for whatever reason. Are we ok with `Comparison not applicable`?

Also, when the difference is 0 do we want to still show  `0%` or the text `No change`. I would have expected the earliest.

Reply:
If there is no change, for **%**, show` 0%` with no icon, and for **raw value** show `No change` with no icon

Reply:
Would kind of expect `0` to appear instead of `no change` even for raw values

Let's just display N/A instead of `comparison not applicable` in the text (it feels quite wordy).

Reply:
`No change` in all cases!

Reply:
From Slack:

- we're going to show `N/A` instead of `comparison not applicable`
- we're going to show `+0%` or `+0` for comparisons where there is no change